tests/utils/test_seed.py

```diff
@@ -0,0 +1,41 @@
+import random
+
+import numpy as np
+import pytest
+import torch
+from numpy import testing
+
+from kale.utils.seed import set_seed
+
+
+@pytest.fixture
+def base_rand():
+    return 0.7773566427005639
+
+
+def test_set_seed_base(base_rand):
+    set_seed()
+    result = random.random()
+    testing.assert_almost_equal(result, base_rand)
```

Comment:
Why use almost_equal? For this situation, you really do want them to test that the random number is exactly the same, otherwise we're not going to get the same outputs.

Reply:
Thanks @drj11, that's better.